app/assets/styles/map.scss

```diff
@@ -8,8 +8,23 @@ $font-choice: Roboto;
 
 #list {
   height: 95%;
+  margin-top: 0px;
   overflow-y: auto;
-  padding: 10px;
+  padding: 0 0.5em 0 0.5em;
```

Comment:
`em` is a relative measure according to the font size and `px` is absolute measure so `em` makes it better for scaling according to the size of things around it instead of arbitrarily choosing `px` sizes that might only work for a certain font/screen size